apps/src/lib/kits/maker/ui/EligibilityConfirmDialog.jsx

```diff
@@ -57,15 +62,21 @@ export default class EligibilityConfirmDialog extends Component {
 
   handleSubmit = () => {
     this.setState({submitting: true});
-    // TODO: extract actual signature and make an ajax POST
-    // fake async for now
-    setTimeout(() => {
-      this.setState({
-        submitting: false,
+    $.ajax({
+     url: "/maker/complete",
+     type: "post",
+     dataType: "json",
+     data: {
+       signature: this.state.signature
+     }
+   }).done(data => {
+     this.props.onSuccess(data.code);
+   }).fail((jqXHR, textStatus) => {
+     this.setState({
+       error: "We're sorry, but something went wrong. Try refreshing the page " +
+        "and submitting again.  If this does not work, please contact support@code.org."
       });
```

Comment:
It looks like we don't clear the `submitting` state in either the success or failure case for this ajax request.  Intentional?

Reply:
In the success case, the dialog is closed. In the failure page, I'd just as soon have them do a full page reload.